server/game/cards/04.1-BotK/SacredSanctuary.js

```diff
@@ -0,0 +1,34 @@
+const ProvinceCard = require('../../provincecard.js');
+
+class SacredSanctuary extends ProvinceCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            title: 'Choose a monk character',
+            when: {
+                onConflictDeclared: (event, context) => event.conflict.conflictProvince === context.source
+            },
+            target: {
+                controller: 'self',
+                cardCondition: card => card.hasTrait('monk'),
+                gameAction: [
+                    ability.actions.ready(),
+                    ability.actions.cardLastingEffect({
+                        condition: () => this.game.isDuringConflict,
```

Comment:
_isDuringConflict_ is a function and needs to be a call like _isDuringConflict()_